static/src/javascripts/projects/commercial/modules/cmp/store.js

```diff
@@ -124,14 +124,12 @@ const getVendorConsentData = (
     cmpVersion: number,
     cookieVersion: number,
     canPersonalise: boolean | null,
-    shortVendorList: ShortVendorList,
-    isRunningCmpCustomise: boolean = false
+    shortVendorList: ShortVendorList
 ): ?VendorConsentData => {
-    if (isRunningCmpCustomise) {
-        log.info('getVendorConsentData: Running Cmp Customise');
-        const cookieVal = readVendorConsentCookie();
-        if (cookieVal) return { ...cookieVal };
-    }
+    log.info('getVendorConsentData: Running Cmp Customise');
```

Comment:
If CMP Customise is going away, we can probably remove this log line

Reply:
👍